### PR TITLE
ClaimReview e2e Tests

### PR DESCRIPTION
**Pull request by caneppelevitor**

### CODE REVIEW

File: server/claim-review/claim-review.service.ts [hunk 1]
```diff
@@ -86,6 +86,7 @@ export class ClaimReviewService {
             { $limit: parseInt(pageSize) }
         );
 
+        // error when uses aggregation
```
Comment (thesocialdev) [inline server/claim-review/claim-review.service.ts:89]: question: the CI didn't file, should it?

Comment (caneppelevitor) [reply server/claim-review/claim-review.service.ts:89]: Maybe I didn't expressed my self well, it doesn't return an error. The problem is that the aggregation return empty  when the test is ran.

File: package.json [hunk 2]
```diff
@@ -127,6 +129,7 @@
     "migrate-mongo-ts": "^1.1.4",
     "mongoose": "^  5.13.15",
     "mongoose-softdelete-typescript": "^0.0.3",
+    "mongodb-memory-server": "^8.15.1",
```
Comment (thesocialdev) [inline package.json:132]: We shouldn't do this. This will download binaries we will never use to the docker image.

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (thesocialdev) [reply package.json:132]: We decided to follow-up on a separate MR

**Review by thesocialdev — APPROVED** [review]

File: server/claim-review/claim-review.service.ts [hunk 3]
```diff
@@ -36,89 +35,69 @@ export class ClaimReviewService {
...
-            lookupClaims(TargetModel.ClaimReview),
-            { $unwind: "$claim" },
-            lookupClaimRevisions(TargetModel.ClaimReview),
-            { $unwind: "$claim.latestRevision" },
-            {
-                $match: {
-                    "claim.isHidden": query.isHidden,
-                    "claim.isDeleted": false,
-                    "claim.nameSpace":
+        const pipeline = this.ClaimReviewModel.find(query)
```
Comment (thesocialdev) [inline server/claim-review/claim-review.service.ts:38]: follow-up: we need to add namespace to claim review to avoid bad populate results

**Pull request merged**